test/views/list-item/content/static.test.js

```diff
@@ -8,7 +8,7 @@ const ContentViewStatic = require('../../../../lib/views/list-item/content/stati
 const assert = require('assert');
 const enzyme = require('enzyme');
 const sinon = require('sinon');
-const React = require('react');
+const React = require('react'); // eslint-disable-line no-unused-vars
```

Comment:
I got eslint failures about react being defined but never used, now it seems to be passing ok.